redis/src/bloom.rs

```diff
@@ -0,0 +1,310 @@
+//! Defines types to use with the bloom filter commands.
+
+#[cfg(feature = "bloom")]
```

Comment:
this is unnecessary in this file, since the import of the file already is guarded by this attribute.

Reply:
👍